projects/plugins/jetpack/modules/related-posts/jetpack-related-posts.php

```diff
@@ -342,96 +342,106 @@ public function get_client_rendered_html_unsupported() {
 	public function render_block_item( $related_post, $block_attributes ) {
 		$instance_id = 'related-posts-item-' . uniqid();
 		$label_id    = $instance_id . '-label';
+		$title       = $related_post['title'];
+		$url         = $related_post['url'];
+		$rel         = $related_post['rel'];
```

Comment:
below we were checking if `$related_post['rel']` wasn't empty; maybe we should do the same here?

Reply:
Do you mean we should do the same for `$title` and `$url`? It's unlikely they'll be empty but I can do it.